external-adapter/src/validator.ts

```diff
@@ -32,20 +34,66 @@ export class Validator {
         }
       }
     } catch (error) {
-      const message = 'Error validating input.'
-      if (error instanceof AdapterError) this.error = error
-      else
-        this.error = new AdapterError({
-          jobRunID: this.validated.id,
-          statusCode: 400,
-          message,
-          cause: error,
-        })
-      logger.error(message, { error: this.error })
-      this.errored = Requester.errored(this.validated.id, this.error)
+      this.throwInvalidInput(error)
     }
   }
 
+  validateOverrides() {
+    if (!this.input.data?.overrides) return
+    try {
+      this.validated.overrides = this.formatOverride(this.input.data.overrides)
+    } catch (e) {
+      this.throwInvalidInput(e)
+    }
+  }
+
+  throwInvalidInput(error: any) {
```

Comment:
This function does not throw, consider renaming.

Reply:
Right